### PR TITLE
Changelog

### PR DESCRIPTION
**Pull request by jburel**
Add Changelog
and bump version ahead of new release

cc @hflynn

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# 3.0.0 (April 2016)
+
+    - flake8 and pep8 compatible
+    - fix 404 error
```
Comment (hflynn) [inline CHANGELOG.md]: Fix (other items have starting capital)

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    - Use valid SPDX license expression 
+
+# 2.0.1 (November 2016)
+
+    - Update Intructions and classifiers
```
Comment (hflynn) [inline CHANGELOG.md]: instructions

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+# 2.0.0 (October 2016)
+
+    - Release the app from PyPI
+    - Re-organise and rename packages
```
Comment (hflynn) [inline CHANGELOG.md]: Re-organize

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+3.0.0 (April 2016)
```
Comment (hflynn) [inline CHANGELOG.md]: 2017

**Comment by jburel** [comment]
@hflynn happy with changes?

**Comment by hflynn** [comment]
Yep, looks good

**Pull request merged**